x-pack/platform/plugins/shared/sample_data_ingest/server/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PluginInitializer, PluginInitializerContext } from '@kbn/core/server';
+import type { SampleDataStartDependencies, SampleDataSetupDependencies } from './types';
+import type { SampleDataIngestConfig } from './config';
+import { SampleDataIngestPlugin } from './plugin';
+export { config } from './config';
+
+export const plugin: PluginInitializer<
+  SampleDataStartDependencies,
+  SampleDataSetupDependencies
+> = async (pluginInitializerContext: PluginInitializerContext<SampleDataIngestConfig>) => {
+  return new SampleDataIngestPlugin(pluginInitializerContext);
+};
```

Comment:
nit: for faster Kibana startup, can we import the plugin code inside the plugin's function?


```suggestion
export { config } from './config';

export const plugin: PluginInitializer<
  SampleDataStartDependencies,
  SampleDataSetupDependencies
> = async (pluginInitializerContext: PluginInitializerContext<SampleDataIngestConfig>) => {
  const { SampleDataIngestPlugin } = await import('./plugin');
  return new SampleDataIngestPlugin(pluginInitializerContext);
};
```